dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/configuration/ListenersConfiguration.java

```diff
@@ -17,33 +17,89 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.batch.core.ExitStatus;
 import org.springframework.batch.core.ItemWriteListener;
+import org.springframework.batch.core.StepExecution;
 import org.springframework.batch.core.listener.StepListenerSupport;
+import org.springframework.batch.core.scope.context.ChunkContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 import uk.ac.ebi.eva.commons.core.models.IVariant;
 import uk.ac.ebi.eva.dbsnpimporter.models.SubSnpCoreFields;
+import uk.ac.ebi.eva.dbsnpimporter.parameters.Parameters;
 
 import java.util.List;
 
 @Configuration
 public class ListenersConfiguration {
 
     @Bean
-    public ItemWriteListener<IVariant> variantsWriteListener() {
-        return new writerListener();
+    public StepListenerSupport<SubSnpCoreFields, IVariant> variantsWriteListener(Parameters parameters) {
+        return new writerListener(parameters);
     }
 
     private static class writerListener extends StepListenerSupport<SubSnpCoreFields, IVariant> {
 
         private static final Logger logger = LoggerFactory.getLogger(writerListener.class);
 
+        private Parameters parameters;
+
+        private long itemsRead;
+
+        public writerListener(Parameters parameters) {
+            this.parameters = parameters;
+            this.itemsRead = 0;
+        }
+
+        @Override
+        public void beforeStep(StepExecution stepExecution) {
+            logger.info("Starting a step");
```

Comment:
Make these messages debug or trace level so we can filter them out in regular runs that don't require such a detailed logging.